helpers/sql/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry Sql Helpers
+
+Todo: Add a description.
```

Comment:
Do you want to add more of a description? I think things are explained well below.